cmd/harmony/main.go

```diff
@@ -803,6 +803,9 @@ func setupConsensusAndNode(hc harmonyconfig.HarmonyConfig, nodeConfig *nodeconfi
 		Uint64("viewID", viewID).
 		Msg("Init Blockchain")
 
+	// Assign GasPriceOracle parameters to the current node
+	currentNode.HarmonyConfig.GasPriceOracle = hc.GasPriceOracle
```

Comment:
I still don't see why this is needed. The `currentNode` stores within itself the entire config (including the `GasPriceOracle`) per below.
https://github.com/harmony-one/harmony/blob/b2e50e694ca7b7cd2adde45724711f9eafff049b/node/node.go#L1043